services/asymkey/commit.go

```diff
@@ -393,6 +393,13 @@ func parseCommitWithSSHSignature(ctx context.Context, c *git.Commit, committerUs
 			log.Error("GetEmailAddresses: %v", err)
 		}
 
+		// committer's noreply email address should be considered as activated
+		// so that user signed with that email address can be verified
+		committerEmailAddresses = append(committerEmailAddresses, &user_model.EmailAddress{
+			IsActivated: true,
+			Email:       committerUser.GetPlaceholderEmail(),
+		})
+
```

Comment:
I don't think the fix is good.

When `parseCommitWithSSHSignature` is called, we already know that `committerUser` belongs to `git commit`. The whole `committerEmailAddresses` logic should be removed.